Makefile.am

```diff
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS=-I m4
 
-SUBDIRS = $(GENERIC_LIBRARY_NAME)
+SUBDIRS = $(GENERIC_LIBRARY_NAME) python
```

Comment:
Why don't we put all this stuff under `swig` instead of `python`, similar to what `hfst` does?

Reply:
hfst has its swig files in `python/` https://github.com/hfst/hfst/tree/master/python, Should I rename python/ to swig/

Reply:
Ah, guess I misremembered. Let's leave it then.